.sass-lint.yml

```diff
@@ -4,6 +4,8 @@ files:
   include: 'frontend/stylesheets/**/*.s+(a|c)ss'
   ignore:
     - 'frontend/stylesheets/components/sweet-alert/**'
+    - 'frontend/stylesheets/components/reveal/**'
+    - 'frontend/stylesheets/presentations.scss'
```

Comment:
I told @RamZallan to do this as:
- We're not going to fix Reveal's CSS
- `reveal.scss` must be imported as such in `presentations.scss` as Reveal ships the SCSS and CSS files in the same folder. Sass-Lint doesn't support line-disable, so this will have to do.